avocado/plugins/spawners/podman.py

```diff
@@ -69,13 +94,19 @@ async def spawn_task(self, runtime_task):
             runtime_task.status = msg
             return False
 
+        test_volume = "--volume=AVOCADO:/tmp"
```

Comment:
I don't understand why this was set here, and then conditionally (because of a task's kind) replaced... care to explain?